### ISSUE TITLE
RTX-KG2 builds with biolink-model.owl.ttl file from the `master` branch rather than a specific versioned branch

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX-KG2#124]

**Comment by ericawood** [comment]
This shouldn't be the case. In all cases where `ont-load-inventory.yaml` is used, it is edited to use the specific versioned `biolink-model.owl.ttl` file:

https://github.com/RTXteam/RTX-KG2/blob/8895f8d13bf90f8a964cdaf3ff24b7b07f78dae5/run-validation-tests.sh#L22-L38
https://github.com/RTXteam/RTX-KG2/blob/8895f8d13bf90f8a964cdaf3ff24b7b07f78dae5/build-multi-ont-kg.sh#L24-L34

**Comment by saramsey** [comment]
Hi @ericawood you are correct (I just tested it to be sure). The following code (lifted from `build-multi-ont-kg.log`) works:
```
sed -i '\@https://raw.githubusercontent.com/biolink/biolink-model/@c\  url: https://raw.githubusercontent.com/biolink/biolink-model/2.1.0/biolink-model.owl.ttl' /home/ubuntu/kg2-code/ont-load-inventory.yaml
```
I think at some point (with all my shenanigans to resume from broken builds, etc.) the *updated* `ont-load-inventory.yaml` file in the local repo got replaced with the GitHub version, reverting back to `master`. That led me to (erroneously) conclude that the in-place edit for that file wasn't happening.
